tests/integration/targets/aci_vrf_fallback_route_group/tasks/main.yml

```diff
@@ -0,0 +1,411 @@
+          - cm_sorted_child_members.2.rnhAddr == "4.4.4.4"
+          - cm_sorted_children.3.fvFBRoute.attributes.fbrPrefix == "1.1.1.1/2"
+          - nm_create_vrf_fbrg is changed
+          - nm_create_vrf_fbrg.previous == []
+          - nm_create_vrf_fbrg.current.0.fvFBRGroup.attributes.name == "test_fallback_route_group"
+          - sorted_children.3.fvFBRoute.attributes.fbrPrefix == "1.1.1.1/2"
+          - sorted_child_members.0.rnhAddr == "1.1.1.1"
+          - sorted_child_members.1.rnhAddr == "2.2.2.2"
+          - sorted_child_members.2.rnhAddr == "4.4.4.4"
+
+    - name: Create vrf fallback route again (idempotency)
+      cisco.aci.aci_vrf_fallback_route_group:
+        <<: *fbrg_info
+      register: nm_create_vrf_fbrg_again
+    
+    - name: Verify that nm_create_vrf_fbrg_again is not changed (idempotency)
+      ansible.builtin.assert:
+        that:
+          - nm_create_vrf_fbrg_again is not changed
+          - nm_create_vrf_fbrg_again.current == nm_create_vrf_fbrg.current
+
+    - name: Create a new vrf Fallback Route Group with minimum required config - Normal mode   
+      cisco.aci.aci_vrf_fallback_route_group:
+        <<: *aci_info
+        tenant: ansible_test
+        vrf: vrf_test
+        name: test_fallback_route_group_2 
+        state: present
+      register: nm_create_vrf_fbrg_2
+
+    - name: Verify that nm_create_vrf_fbrg_2 is created with correct attributes
+      ansible.builtin.assert:
+        that:
+          - nm_create_vrf_fbrg_2 is changed
+          - nm_create_vrf_fbrg_2.previous == []
+          - nm_create_vrf_fbrg_2.current.0.fvFBRGroup.attributes.name == "test_fallback_route_group_2"
+          - nm_create_vrf_fbrg_2.current.0.fvFBRGroup.children is not defined
+
+    - name: Create a new vrf Fallback Route Group with minimum required config - Check mode  
+      cisco.aci.aci_vrf_fallback_route_group:
+        <<: *aci_info
+        tenant: ansible_test
+        vrf: vrf_test
+        name: test_fallback_route_group_2 
+        state: present
+      check_mode: true
+      register: cm_create_vrf_fbrg_2
+
+    - name: Verify that nm_create_vrf_fbrg_2 is created with correct attributes
+      ansible.builtin.assert:
+        that:
+          - cm_create_vrf_fbrg_2.current.0.fvFBRGroup.attributes.name == "test_fallback_route_group_2"
+          - cm_create_vrf_fbrg_2.current.0.fvFBRGroup.children is not defined
+    
+    - name: Update vrf_fallback_route - Check mode
+      cisco.aci.aci_vrf_fallback_route_group:
+        <<: *aci_info
+        tenant: ansible_test
+        vrf: vrf_test
+        name: test_fallback_route_group
+        fallback_route: 1.1.1.1/2
+        fallback_members: 
+          - 1.1.1.1
+          - 2.2.2.2
+          - 4.4.4.4
+          - 5.5.5.5
+        description: Test for Fallback Route Group
+        state: present
+      check_mode: true
+      register: cm_update_vrf_fbrg
+
+    - name: Update vrf_fallback_route - Normal mode
```

Comment:
why not test the update of all the things (description, route) that are allowed to be updated?